include/aspect/fe_variable_collection.h

```diff
@@ -198,6 +198,11 @@ namespace aspect
        */
       const FEVariable<dim> &variable(const std::string &name) const;
 
+      /**
```

Comment:
Can you update the documentation of the function above so that it mentions it will return the first variable of that name?